RealtimeStreamToImpala/README.rst

```diff
@@ -0,0 +1,54 @@
+Realtime Stream To Impala Adapter Configuration
```

Comment:
Take a look at other guides - you may want to follow similar format. It has getting CDAP SDK part, running shell, etc.